### PR TITLE
Add Heroku support

### PR DESCRIPTION
**Pull request by mihaiparvu**
Makes Heroku users life easier.

### CODE REVIEW

**Comment by robobenklein** [comment]
Other people look at programmers and wonder why we pushed a full continuous delivery pipeline with production deployment just to farm a broken html5 minigame.

Programmers see this and wonder why we haven't yet created a Kubernetes cluster config for massively parallel cheating via service automation.

File: README.md [hunk 1]
```diff
@@ -67,3 +67,16 @@ php cheat.php token2
 2. To build: `docker build . -t steamdb/saliencheat`
 3. To run: `docker run -it --init --rm -e TOKEN=<32 character token from gettoken url> steamdb/saliencheat`
 4. To stop running, Ctrl+C
+
+## Heroku
+1. Fork this repository on your GitHub account.
```
Comment (xPaw) [inline README.md]: Is the fork really necessary?

Comment (robobenklein) [reply README.md]: In order for the connection to happen, Heroku only pulls from owned or membered repos under the user. (Such is the Oauth scopes method they use.)

Comment (mihaiparvu) [reply README.md]: Yep, repo admin access required.

Comment (JourneyOver) [reply README.md]: Believe it or not this actually isn't required if you set up a one-click heroku deploy button like I have in PR #190

Comment (mihaiparvu) [reply README.md]: Nice job @JourneyOver!

**Pull request merged**